src/UnityPackage/Assets/Editor/ServerSettings/ChainsafeServerSettings.cs

```diff
@@ -43,6 +45,12 @@ void Awake()
             PlayerPrefs.Save();
         }
 
+        if (Token == ("Please Enter Your Chain's Native Token i.e Eth, Cro") && (PlayerPrefs.GetString("Token") != ""))
```

Comment:
I think the terminology here is symbol, consider rephrasing to: 
```suggestion
        if (Symbol == ("Please Enter Your Chain's Native Symbol i.e Eth, Cro") && (PlayerPrefs.GetString("Symbol") != ""))
```

Reply:
like that interms of indentation and symbol?

Reply:
@sneakzttv I mean the term for the chain's local token is usually referred to as "Symbol"

For example, the fields in Metamask
![image](https://github.com/ChainSafe/web3.unity/assets/10461622/0a5a126f-5798-401a-b30c-56dd1ee9d509)

Reply:
yah got it, updated the rest, left the object as is